src/style.css

```
@@ -1,3 +1,80 @@
+html {
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+h1 {
+  text-align: center;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+header {
```

Use the class selector, fix it everywhere
```suggestion
.header {
```